### DISCUSSION TITLE
从零到一：如何构建一个基于知识图谱的智能问答助手？ - siwei.io

### DISCUSSION THREAD

**Discussion by giscus** [siwei-io/siwei-io.github.io#24] [Comments]
# 从零到一：如何构建一个基于知识图谱的智能问答助手？ - siwei.io

如何利用图数据库从0-1构建一个特定领域问答助手？本文手把手带你构建一个简易版的篮球领域智能问答机器人。

https://siwei.io/siwi/

**Comment by ghost** [comment]
有考虑做个接入chatgpt api的demo吗，用于知识对话和节点智能扩展...

**Reply by wey-gu** [reply]
👍🏻，我最近其实在想，等过了这两周之后，可以做一下~~

此外我其实做了一些 gpt 和知识图谱的、NebulaGraph 东西，还写了书稿还没发表，不过还比较简单，

- https://github.com/wey-gu/simple-graph-render
- https://github.com/wey-gu/NebulaGraph-GPT

**Reply by wey-gu** [reply]
如果你感兴趣的话，我们也可以一起搞哈~~

**Reply by ghost** [reply]
原来已经有两个项目了，去学习下，我技术一般，最近都是用new bing搜了做的
一起搞，怎么协作啊，我可以帮实现一些简单的todo功能

最近的一个应用场景就是在学习 aigc，但是找的开源模型效果一般，然后论文也看不明白，资料太多了，所以希望有个概念图类似的东西，把各种方法和改进之间的关系梳理清楚
